contrib/IECoreUSD/src/IECoreUSD/DataAlgo.cpp

```diff
@@ -92,6 +94,15 @@ GeometricData::Interpretation interpretation( TfToken role )
 	return GeometricData::None;
 }
 
+static const bool g_forceAssetPathForwardSlash = []() -> bool {
```

Comment:
I'd like to limit this to Windows only, so there's no danger of anything changing on Linux.